src/Task_3/Switch_with_places.java

```diff
@@ -0,0 +1,28 @@
+package Task_3;
+
+import java.util.Scanner;
+
+public class Switch_with_places {
+
+    public static void main(String args[]){
+
+        Scanner readeri = new Scanner(System.in);
+        System.out.println("Enter i: ");
+        int i = readeri.nextInt();
+
+        Scanner readerj = new Scanner(System.in);
+        System.out.println("Enter j: ");
+        int j = readerj.nextInt();
+
+        int temp;
+
+        temp = i;
```

Comment:
Can you think of a way to switch the two variables without introducing the third one?